docs/book/src/multiversion-tutorial/testdata/project/dist/chart/templates/prometheus/monitor.yaml

```diff
@@ -15,6 +15,7 @@ spec:
       bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
       tlsConfig:
         {{- if .Values.certmanager.enable }}
+        serverName: project-controller-manager-metrics-service.{{ .Release.Namespace }}.svc
```

Comment:
Hi @abhishekdwivedi3060 ^ 
We also missed update the helm-chart

Reply:
Same here. `test-e2e-local` didn't catch this.
Is it something we can add in our test suite?

Reply:
We test the Helm chart, but we currently have no test confirming whether the Helm chart's results match the Golang implementation. I'm not sure how we could properly validate this.

Additionally, we verify the serverName integration with Prometheus through end-to-end (e2e) tests.

If you have any ideas or suggestions, feel free to open a PR with your recommendations!